test/parallel/test-buffer-pending-deprecation-map.js

```diff
@@ -0,0 +1,9 @@
+// Flags: --pending-deprecation --no-warnings
+'use strict';
+
+const common = require('../common');
+const Buffer = require('buffer').Buffer;
+
+process.on('warning', common.mustNotCall((warning) => {}));
+
+Buffer.alloc(10).map((i) => {});
```

Comment:
Note: it won't hurt to test other methods here, e.g. `Buffer.alloc(10).filter(() => true)`, which also would have been affected if not for the fix above.